### PR TITLE
[3.9] hardcode examples content version to 3.9

### PR DESCRIPTION
**Pull request by vrutkovs**
This should fix gcp job

### CODE REVIEW

**Comment by sdodson** [comment]
/lgtm
thanks for getting to this!

File: roles/openshift_hosted_templates/defaults/main.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 hosted_base: "{{ openshift.common.config_base if openshift_is_containerized | bool else '/usr/share/openshift' }}/hosted"
 hosted_deployment_type: "{{ 'origin' if openshift_deployment_type == 'origin' else 'enterprise' }}"
 
-content_version: "{{ openshift.common.examples_content_version }}"
+content_version: "3.9"
```
Comment (sdodson) [inline roles/openshift_hosted_templates/defaults/main.yml]: looks like it needs to be 'v3.9' instead.

**Comment by sdodson** [comment]
Updated openshift_examples too. Should be good to go now.

**Comment by sdodson** [comment]
I can't figure out why it would've gotten v3.10 there.
/retest

**Comment by vrutkovs** [comment]
/hold

This is insufficient, GCP job still picks up 3.10 RPMs

**Comment by vrutkovs** [comment]
Nope, that's not gonna work. Versioned repo should be used instead.

**Comment by sdodson** [comment]
> Nope, that's not gonna work. Versioned repo should be used instead.

Just coming back to this, did we do that?

**Comment by vrutkovs** [comment]
Yes, in CI we're injecting http://mirror.centos.org/centos/7/paas/x86_64/openshift-origin39/, which has no 3.10 packages

**Pull request closed without merging**